.travis.yml

```diff
@@ -2,4 +2,6 @@ language: nix
 before_script:
   - sudo mkdir /etc/nix && echo 'sandbox = true' | sudo tee /etc/nix/nix.conf
 script:
-  - nix-build --quiet release.nix -A build.x86_64-linux -I nixpkgs=channel:nixos-19.09
+  - nix-build -I nixpkgs=channel:nixos-19.09 --quiet release.nix -A build.x86_64-linux
+  # Run black from the latest release channel and ensure no diff
+  - nix run -I nixpkgs=channel:nixos-20.03 nixpkgs.python3Packages.black -c black --check .
```

Comment:
CC @AmineChikhaoui @adisbladis @grahamc I don't have much experience with Travis and don't think I can test this directly, but the cmd given here works locally for me.